curriculum/challenges/english/25-front-end-development/lab-factorial-calculator/66c07238b01053abaf812065.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 66c07238b01053abaf812065
+title: Build a Factorial Calculator
+challengeType: 14
+dashedName: build-a-factorial-calculator
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
```

Comment:
We should add a sentence or two explaining what a factorial is because a lot of campers might not know or remember from math class years ago